lib/capifony_symfony1.rb

```diff
@@ -54,7 +54,7 @@ def guess_symfony_orm
 def guess_symfony_lib
   symfony_version = capture("cd #{latest_release} && #{php_bin} ./symfony -V")
 
-  /\((.*)\)/.match(symfony_version)[1]
+  /1\.[0-1]\.[0-9]{2}/.match(symfony_version)[1]
```

Comment:
@willdurand `php ./symfony -V` output (on 1.0.22) is : `symfony: version 1.0.22`